_guides/getting_started.md

```diff
@@ -126,113 +231,155 @@ best constant = 0.500000
+
+>**Note:** Always use a different data-set for testing vs. training for real prediction settings.
 
-## Auditing
+## Auditing machine learning applications with Vowpal Wabbit
 
-When developing a new ML application, it's very helpful to debug. VW can help with this using the `--audit` option, which outputs extra informations about predictions and features.
+VW has a built-in audit option that helps debug machine learning (ML) applications.
+
+**Use** `--audit` to output helpful information about predictions and features. For example: 
 
 ```sh
 vw house_dataset --audit --quiet
 ```
 
-Output:
+The output for this command is:
+
 ```
 0
-  price:229902:0.23:0@0  sqft:162853:0.25:0@0  age:165201:0.05:0@0  2006:2006:1:0@0  Constant:116060:1:0@0
+  price:229902:0.23:0@0 sqft:162853:0.25:0@0 age:165201:0.05:0@0 2006:2006:1:0@0 Constant:116060:1:0@0
 0 second_house
-  price:229902:0.18:0@0  sqft:162853:0.15:0@0  age:165201:0.35:0@0  1976:1976:1:0@0  Constant:116060:1:0@0
+  price:229902:0.18:0@0 sqft:162853:0.15:0@0 age:165201:0.35:0@0 1976:1976:1:0@0 Constant:116060:1:0@0
 1 third_house
-  price:229902:0.53:0.882655@0.2592  age:165201:0.87:0.453833@0.98  sqft:162853:0.32:1.05905@0.18  Constant:116060:1:0.15882@8  1924:1924:1:0@0
+  price:229902:0.53:0.882655@0.2592 age:165201:0.87:0.453833@0.98 sqft:162853:0.32:1.05905@0.18 Constant:116060:1:0.15882@8 1924:1924:1:0@0
 ```
 
-Every example uses two lines. The first line has the prediction, and the second line has one entry per feature. Looking at the first feature, we see:
+Every example uses two lines:
+
+The first line is the prediction.
+The second line shows one entry per feature. 
+
+VW has an advanced_namespaces_ feature that allows you to group features and operate them on-the-fly. Namespace options include the following: 
+
+Use `-q XY` to cross a pair of namespaces.
+Use `--cubic XYZ` to cross 3 namespaces.
+Use `--lrq XYn` to low-rank quadratic interactions.
+Use `--ignore X` to skip all features belonging to a namespace.
+
+The first feature listed is:
 
 ```
 price:229902:0.23:0@0.25
 ```
 
-- `price` is the original feature name. If you use a namespace, it appears before `^` (i.e. `Namespace^Feature`). Namespaces are an advanced feature which allows you to group features and operate them on-the-fly, in the core of VW with the options: `-q XY` (cross a pair of namespaces), `--cubic XYZ` (cross 3 namespaces), `--lrq XYn` (low-rank quadratic interactions), and `--ignore X` (skip all features belonging to a namespace).
-- `229902` is the index of the feature, computed by a hash function on the feature name.
-- `0.23` is the value of the feature.
-- `0` is the value of the feature's weight.
-- `@0.25` represents the sum of gradients squared for that feature when you are using per-feature adaptive learning rates.
+For the first feature listed, **`price`** is the original feature name. If you use a namespace, it appears before `^`. For example: 
+
+`Namespace^Feature` 
```

Comment:
This looks really confusing as it breaks the bullet point description of the line

Reply:
Yes, it does, updated and rearranged. Please check for accuracy.